packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,12 +93,14 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
           aria-label={ label }
+          aria-describedby={ id + '-option' }
           aria-activedescendant={ selectedIndex >= 0 ? (this._id + '-list' + selectedIndex) : (this._id + '-list') }
           >
           <span
+            id={ id + '-option' }
             className='ms-Dropdown-title'
             key={ selectedIndex }
-            aria-live={ disabled || isOpen ? 'off' : 'polite' }
+            aria-live={ disabled || isOpen ? 'off' : 'assertive' }
```

Comment:
Not sure of the behavior with JAWS but just did a quick manual test with NVDA and Narrator:
1. The option changes are not announced at all on IE, EDGE or Firefox. (assertive or polite)
    Seems like an issue with aria-live span, I guess re-rendering of live-regions is not detected by screen readers. In this case, React re-renders the live-region span on option change.
Is it working for you? Can you specify screen reader and browser version?
2. Moving `aria-live` attributes to parent (ms-Dropdown) works as expected.